common/interpreter/wawaka_wasm/WasmExtensions.cpp

```diff
@@ -37,70 +37,16 @@
 
 namespace pe = pdo::error;
 
-/* ----------------------------------------------------------------- *
```

Comment:
Did we had to change order of method declaration?  Makes diff pretty useless ... :(

Reply:
yes. there are a number of functions that were just removed. pretty much everything else had to change.